packages/mobile/src/transactions/saga.ts

```diff
@@ -98,6 +112,53 @@ export function* sendAndMonitorTransaction<T>(
   }
 }
 
+export function* fetchRecentTxRecipients() {
```

Comment:
I tend to use `fetch` for things that go to network. How about `refresh` or `find`?